Applications/Games/Subnautica/Steam/script.js

```diff
@@ -15,21 +15,24 @@ var installerImplementation = {
             .author("Zemogiter")
             .applicationHomepage("https://unknownworlds.com/subnautica/")
             .wineDistribution("upstream")
-            .wineVersion("4.0-rc1")
+            .wineVersion("4.0")
             .wineArchitecture("amd64")
             .appId(264710)
             .preInstall(function (wine, wizard) {
+                wizard.message(tr("Please ensure you have the latest drivers (418.30 minimum for NVIDIA and mesa 19 for AMD) or else this game might not work correctly."));
+                wizard.message(tr("You can make the game smoother by using this: https://github.com/lutris/lutris/wiki/How-to:-Esync"));
                 wine.vcrun2013();
                 wine.corefonts();
                 wine.d3dx11();
                 wine.DXVK();
```

Comment:
Do yo still needs d3d11 verb with DXVK (here it is working because the DXVK verb is after the d3d11)?

Reply:
I did not test it with just dxvk. I've tested it the other way around (d3dx11 without dxvk) when the game failed to display a window when using shader mods.

Reply:
Please test without d3d11.